### PR TITLE
Fix invalid exception for class method with name "get"

### PR DESCRIPTION
**Pull request by KaruroChori**
From here: https://github.com/bellard/quickjs/pull/258

Fixing issue mentioned in https://github.com/saghul/txiki.js/issues/511#issuecomment-2096231405

### CODE REVIEW

**Review by saghul — APPROVED** [review]

File: quickjs.c [hunk 1]
```diff
@@ -20810,8 +20810,8 @@ static int __exception js_parse_property_name(JSParseState *s,
             if (next_token(s))
                 goto fail1;
             if (s->token.val == ':' || s->token.val == ',' ||
-                s->token.val == '}' || s->token.val == '(') {
-                is_non_reserved_ident = TRUE;
+                s->token.val == '}' || s->token.val == '(' ||
+                s->token.val == '=' ) {is_non_reserved_ident = TRUE;
```
Comment (chqrlie) [inline quickjs.c]: Missing line break.
Suggested style:
```
                s->token.val == '}' || s->token.val == '(' ||
                s->token.val == '=' ) {
                    is_non_reserved_ident = TRUE;
```

**Comment by KaruroChori** [comment]
The indentation of the code I copied was a bit wonky so I fixed it, and more importantly I forgot to remove a line from the test suite. This time it should be ok :/.

File: tests/test_language.js [hunk 2]
```diff
@@ -336,6 +336,10 @@ function test_class()
     assert(S.x === 42);
     assert(S.y === 42);
     assert(S.z === 42);
+    class P {
+        get = () => "123"
+      }
+      assert(new P().get() === "123");
```
Comment (chqrlie) [inline tests/test_language.js]: Code is not aligned.

**Review by chqrlie — CHANGES_REQUESTED** [review]

File: quickjs.c (hunk 1 above)
Comment (KaruroChori) [reply quickjs.c]: Already solved, it was copied as it was over from the source material but later fixed.

**Review by chqrlie — APPROVED** [review]

**Pull request merged**